ensemble-dss/build.gradle

```diff
@@ -22,8 +21,8 @@ repositories {
 }
 dependencies {
     implementation project(":FIRO_TSEnsembles")
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.2'
     implementation ('mil.army.usace.hec:hec-monolith:3.0.0-rc5'){
```

Comment:
Consider updating hec-monolith. I suggest using the same as the latest Beta DssVue :   hec-monolith:3.3.23

Reply:
Updated to suggested monolith version